nursery/unmanaged-call-dotnet.yml

```diff
@@ -0,0 +1,10 @@
+rule:
+  meta:
+    name: unmanaged call dotnet
```

Comment:
would be helpful to add a description of what this means

Reply:
updated w/ description